src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/tdd/autodelete-true/models/default.metadata.json

```diff
@@ -0,0 +1,9097 @@
+{
```

Comment:
Tests should run  ok wiithout including this file

Reply:
removed